nodepool/elements/jenkins-slave/extra-data.d/20-jenkins-user

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+# the location to fetch the public keys from
+SSH_PUBLIC_KEYS_URL="https://raw.githubusercontent.com/rcbops/rpc-gating/master/keys/rcb.keys"
+
+# fetch the public keys and put them inside the chroot
+curl --connect-timeout 5 \
+     --retry 3 \
+     ${SSH_PUBLIC_KEYS_URL} > ${TMP_HOOKS_PATH}/ssh-public-keys
```

Comment:
We assume TMP_HOOKS_PATH is defined in the env (??)

Reply:
``TMP_HOOKS_PATH`` is a standard path provided by diskimage-builder. It provides a hook path for the extra-data to get something from the host into the chroot. See https://github.com/openstack/diskimage-builder/blob/221c44f24cf398247c9b94e822965b5f6f048f11/doc/source/developer/developing_elements.rst#phase-subdirectories

Reply:
👍